run.sh

```diff
@@ -120,6 +117,106 @@ function parse_ports() {
     done
 }
 
+function deploy_ear() {
+    
+    ssh_key=$(cat ${HOME}/.ssh/id_rsa.pub)
+    grep $ssh_key ~/.ssh/authorized_keys >/dev/null 2>&1
+    if [ $? -ne 0 ] ; then
+      cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
```

Comment:
This will fail if there is no _id_rsa.pub_ available, which crashes the run.sh script. We shouldn't force this as it may break alot of dev environments, testing etc.

Reply:
Fixed.